romana-install/roles/provision/create-aws-hosts/templates/aws_cf_small_cluster.json

```diff
@@ -615,15 +118,10 @@ under the License.
             ]
           }
         }
-      },
-      "Metadata": {
-        "AWS::CloudFormation::Designer": {
-          "id": "4b80e302-c4a6-42ad-92e2-90b08b10f870"
-        }
       }
     },
-{% for node in cf_data.compute_hosts[:compute_nodes] %}
-    "{{ node }}": {
+{% for _ in range(compute_nodes) %}
+    "Compute{{ '%02d' | format(loop.index) }}": {
```

Comment:
This is a Jinja template, right? You should be able to declare a variable, like so:

    {% set node_name = 'Compute{{ '%02d' | format(loop.index)' %}

You could then just refer to {{ node_name }} here and in line 150.

Haven't tried that much, but I think that should be possible.